training_record/services.py

```diff
@@ -417,7 +422,7 @@ def get_number_of_records_without_feedback(user):
 class CampusEventFeedbackService:
     '''Provide services for CampusEventFeedback.'''
     @staticmethod
-    def create_feedback(record, content):
+    def create_feedback(user, record, content):
```

Comment:
Use `context['user']` instead, these variables are not related to the business logic, putting it here looks strange.